src/api-request-handler.ts

```diff
@@ -0,0 +1,66 @@
+import axios, { AxiosInstance } from 'axios';
+
+import {
+    InsufficientCreditsError,
+    InvalidParameterError,
+    InvalidStateError,
+    RevAiApiError
+} from './models/RevAiApiError';
+
+export type HttpMethodTypes = 'post' | 'get' | 'delete';
+export type AxiosResponseTypes = 'stream' | 'json' | 'text';
```

Comment:
Is this type already defined within Axios?

Reply:
I don't want to duplicate the definition.

Reply:
It is not unfortunately